map.ipynb

```diff
@@ -0,0 +1,386 @@
+    ")\n",
+    "add_point.on_click(handle_add)\n",
+    "\n",
+    "remove_point = Button(\n",
+    "    description='Remove Point(s)',\n",
+    "    button_style='danger',\n",
+    "    disabled=False,\n",
+    "    tooltip='Click to remove selected point(s) from the list.',\n",
+    ")\n",
+    "remove_point.on_click(handle_remove)\n",
+    "\n",
+    "start_date = date_widget('Start Date:')\n",
+    "end_date = date_widget('End Date:')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "m.add_layer(p)\n",
+    "m.add_layer(f)\n",
+    "m.add_layer(c)\n",
+    "m.add_layer(marker_group)\n",
+    "\n",
+    "m += layer_control\n",
+    "\n",
+    "m.on_interaction(handle_click)\n",
+    "m.add_control(legend)\n",
+    "\n",
+    "box_layout = Layout(display='flex',\n",
+    "                flex_flow = 'column', \n",
+    "                width='110%',\n",
+    "                align_items = 'center')\n",
+    "\n",
+    "control_box = Box(children = [start_date, end_date, latitude, longitude, add_point, point_list, remove_point, run], layout=box_layout)\n",
+    "\n",
+    "AppLayout(center = m, \n",
+    "    right_sidebar = control_box,\n",
+    "    align_items = 'center')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Starts RVIC process\n",
+    "url = build_url(start_date.value, end_date.value, points)\n",
+    "input_response = requests.get(f'http://docker-dev03.pcic.uvic.ca:30111/osprey/input?{url}').content\n",
+    "print(input_response.decode('utf-8'))\n",
+    "status_url = input_response.split()[-1].decode('utf-8')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Check status of RVIC process\n",
+    "status_response = requests.get(f'http://docker-dev03.pcic.uvic.ca:30111{status_url}').content\n",
+    "print(status_response.decode('utf-8'))\n",
+    "\n",
+    "while 'Process is still running.' in status_response.decode('utf-8'):\n",
+    "    time.sleep(5)\n",
+    "    status_response = requests.get(f'http://docker-dev03.pcic.uvic.ca:30111{status_url}').content\n",
+    "    print(status_response.decode('utf-8'))\n",
+    "    \n",
+    "output_url = status_response.split()[-1].decode('utf-8')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = requests.get(f'http://docker-dev03.pcic.uvic.ca:30111{output_url}').content"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Saves data as NetCDF data set\n",
+    "with tempfile.NamedTemporaryFile(dir=os.getenv('TMPDIR', default='/tmp/')) as fp:\n",
+    "    fp.write(output)\n",
+    "    data = netCDF4.Dataset(fp.name) "
```

Comment:
I recommend also printing `data.variables`, and the values of the `streamflow` variable as additional validation checks. You can get the variable values by using `data["streamflow"][:]`.